dashboard.py

```diff
@@ -735,17 +712,15 @@ def main() -> None:
     df = pandas.read_sql(
         "select * from test_run where platform = 'linux' "
         "and call_outcome in ('passed', 'failed')"
-        "and coiled_runtime_version >= '0.2.1' ",
+        "and coiled_runtime_version = 'upstream'",
```

Comment:
I must have missed this, is it hardcoded somewhere?
Also, as mentioned I think we need to preserve the distintion between latest and upstream.
Finally, I don't think this will work with A/B tests.